src/System.Diagnostics.Process/tests/System.Diagnostics.Process.Tests/Given_ProcessStartInfoType.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Xunit;
+
+namespace System.Diagnostics.ProcessTests
+{
+    public class Given_ProcessStartInfoType : ProcessTestBase
+    {
+        [Fact]
+        public void Then_EnvironmentShouldBe()
```

Comment:
This is a super long test that seems like it'll be difficult to debug if (when) something breaks. Could this be broken up into more cases to help with debug-ability?